src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/MetricsStore.cs

```diff
@@ -165,10 +165,10 @@ public async Task SnapshotMetrics(Stream outputStream, CancellationToken token)
 
         private static string GetMetricLabels(ICounterPayload metric, double? quantile)
         {
-            string metadata = metric.Metadata;
+            string allMetadata = metric.CombineTags();
```

Comment:
So for prometheus export, we will combine meter/instrument/instance tags all into one?

Reply:
That's the current behavior, but if we aren't sure we want to put all the tags together we could display them differently (not sure what that would look like for PEF), or omit them temporarily if we're concerned about future breaking changes.